src/tokens/transformation/ios/index.js

```diff
@@ -6,3 +11,61 @@ module.exports = {
     'ios/fontStyles': require('./fontStyles')
   }
 }
+
+StyleDictionary.registerFormat({
+  name: 'ios/gradients',
+  formatter: ({ dictionary, options, file }) => {
+    const template = _template(
+      fs.readFileSync(__dirname + '/templates/Gradients.swift.template')
+    )
+
+    const gradients = {
+      linear: dictionary.allTokens
+        .filter(
+          (token) =>
+            token.type === 'custom-gradient' &&
+            token.value.gradientType === 'linear'
+        )
+        .map((token) => {
+          token.name = changeCase.camelCase(token.path.pop())
+          return token
+        })
+    }
+
+    return template({ gradients, options, file })
+  }
+})
+
+StyleDictionary.registerFormat({
+  name: 'ios/colorSetAccessors',
+  formatter: ({ dictionary, options, file }) => {
+    const template = _template(
+      fs.readFileSync(__dirname + '/templates/ColorSetAccessors.swift.template')
+    )
+
+    var colors = {}
+    dictionary.allTokens
+      .filter((token) => token.type === 'color')
+      .forEach((token) => {
+        if (token.name === 'ColorBlack' || token.name === 'ColorWhite') {
+          // The system already provides these and will conflict with the names
+          return
+        }
+        const name = changeCase.snakeCase(
+          token.path
+            .filter((i) => {
+              return i != 'dark' && i != 'light' && i != 'color'
```

Comment:
Nit: `!=` ==> `!==`

Optional: convert to inline arrow function

Reply:
Fixed! Though the formatter forced it to include the parenthesis for the arg and the conditions onto a separate line